src/transactions/utils.ts

```diff
@@ -1,33 +1,44 @@
 import i18n from 'src/i18n'
+import { TokenTransaction } from 'src/transactions/types'
 import { formatFeedSectionTitle, timeDeltaInDays } from 'src/utils/time'
 
 // Groupings:
-// Recent -> Last 7 days.
+// Recent -> Last 7 days (standby transactions always at the top, followed by recent confirmed transactions).
 // [Current month] - "July" -> Captures transactions from the current month that aren’t captured in Recent.
 // [Previous months] - "June" -> Captures transactions by month.
 // [Months over a year ago] — "July 2019" -> Same as above, but with year appended.
 // Sections are hidden if they have no items.
-export function groupFeedItemsInSections<T extends { timestamp: number }>(items: T[]) {
+export function groupFeedItemsInSections(
+  standbyTransactions: TokenTransaction[],
+  confirmedTransactions: TokenTransaction[]
+) {
   const sectionsMap: {
     [key: string]: {
-      data: T[]
+      data: TokenTransaction[]
```

Comment:
Just curious, ee can remove the paramterized type because we remove the legacy code?

Reply:
yes i think so! :D